Samples/macOS-Swift/macOS-Swift.xcodeproj/project.pbxproj

```diff
@@ -211,27 +427,127 @@
 			buildRules = (
 			);
 			dependencies = (
+				84B5A9ED2DBD79E7003327C9 /* PBXTargetDependency */,
 				6308534D2440C5F500DDE4CE /* PBXTargetDependency */,
 			);
 			name = "macOS-Swift";
 			productName = "macOS-Swift";
 			productReference = 637AFDEB243B04320034958B /* macOS-Swift.app */;
 			productType = "com.apple.product-type.application";
 		};
+		84B5A8F32DBD1A86003327C9 /* macOS-Swift-Other */ = {
```

Comment:
This new target is necessary for having two different macOS apps in ui tests to verify isolation of stored data